slay

```diff
@@ -135,8 +135,16 @@ if brew list | grep -Fq brew-cask; then
 fi
 
 ###############################################################################
-# INSTALL: brews
+# INSTALL: taps & brews
 ###############################################################################
+if [ -e $cwd/swag/taps ]; then
+	chapter "🍺 Tapping some casks for Homebrew…"
+
+	for tap in $(<$cwd/swag/taps); do
+	    tap_cask $tap
```

Comment:
Where is `tap_cask` defined?

Reply:
It might be replaceable with simply `brew tap $tap` (looking at [this example](https://devcenter.heroku.com/articles/heroku-cli#download-and-install)).